scripts/generate_test_suites.py

```diff
@@ -0,0 +1,39 @@
+import os
+import sys
+
+parts = sys.argv[1]
+
+instrumentation_tests_path = 'instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests'
+core_tests = os.listdir(f'{instrumentation_tests_path}/core')
+ui_tests = os.listdir(f'{instrumentation_tests_path}/ui')
+
+core_part_size = int(len(core_tests) / int(parts)) + 1
+
+suites = {}
+for index, test in enumerate(core_tests):
+	part_number = int(index / core_part_size) + 1
```

Comment:
You'll never get 0 here, is it expected?

Reply:
yes, start count from 1 is more human way to display it in circle ui